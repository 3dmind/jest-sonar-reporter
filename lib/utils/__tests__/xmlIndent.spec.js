'use strict';

const xmlIndent = require('../xmlIndent');

describe('xmlIndent', () => {
  test('should generate spaces', () => {
    const indent = xmlIndent(4);
    expect(indent).toMatchSnapshot();
  });
});
