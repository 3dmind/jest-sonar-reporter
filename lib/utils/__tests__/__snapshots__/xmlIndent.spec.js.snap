// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`xmlIndent should generate spaces 1`] = `"    "`;
